static/elements/chromedash-metadata.js

```diff
@@ -154,6 +154,7 @@ class ChromedashMetadata extends LitElement {
 
   _processResponse(response) {
     // TODO(ericbidelman): Share this data across instances.
+    console.log(response);
```

Comment:
I don't think this belongs in this PR.  Please revert this file to have a more focused PR.

Reply:
My bad. Its done btw. Sorry for the inconvenience.